### PR TITLE
CLDIDE-2252

### PR DESCRIPTION
**Pull request by sleshchenko**

### CODE REVIEW

File: codenvy-api-account/src/main/java/com/codenvy/api/account/shared/dto/AccountMetrics.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import com.codenvy.dto.shared.DTO;
+
+import java.util.List;
+
+/**
+ * @author Sergii Leschenko
+ */
+@DTO
+public interface AccountMetrics {
+    Boolean isPremium();
```
Comment (vzhukovskii) [inline codenvy-api-account/src/main/java/com/codenvy/api/account/shared/dto/AccountMetrics.java]: javadocs ?

File: codenvy-api-account/src/main/java/com/codenvy/api/account/shared/dto/WorkspaceMetrics.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+package com.codenvy.api.account.shared.dto;
+
+import com.codenvy.dto.shared.DTO;
+
+/**
+ * @author Sergii Leschenko
+ */
+@DTO
+public interface WorkspaceMetrics {
+    String getWorkspaceId();
```
Comment (vzhukovskii) [inline codenvy-api-account/src/main/java/com/codenvy/api/account/shared/dto/WorkspaceMetrics.java]: javadocs ?

File: codenvy-api-account/src/main/java/com/codenvy/api/account/server/AccountService.java [hunk 3]
```diff
@@ -1080,14 +1081,39 @@ public void redistributeResources(@ApiParam(value = "Account ID", required = tru
...
+     *         account id
+     */
+    @ApiOperation(value = "Get size of consumed memory during current billing period by account",
+                  notes = "Returns size of consumed memory during current billing period by account. Roles: account/owner, system/admin.",
+                  position = 17)
+    @ApiResponses(value = {
+            @ApiResponse(code = 204, message = "OK"),
+            @ApiResponse(code = 403, message = "Access denied"),
+            @ApiResponse(code = 404, message = "Not found"),
+            @ApiResponse(code = 409, message = "Conflict Error"),
```
Comment (mshaposhnik) [inline codenvy-api-account/src/main/java/com/codenvy/api/account/server/AccountService.java]: No conflict exception throws here

**Comment by sleshchenko** [comment]
commit moved into branch meterBased https://github.com/codenvy/platform-api/commit/6d299bcdc3dc7b1b5a98149efd88cc9113d83a88

**Pull request closed without merging**